src/rust/engine/async_latch/src/lib.rs

```diff
@@ -72,7 +72,7 @@ impl AsyncLatch {
     // To see whether the latch is triggered, we clone the receiver, and then wait for our clone to
     // return None, indicating that the Sender has been dropped.
```

Comment:
Mind updating this?

Reply:
I missing context. In what way?

Reply:
"then wait for our clone to return None" is no longer accurate.

Reply:
Fixed.